### DISCUSSION TITLE
Lockbit ransomware

### DISCUSSION THREAD

**Discussion by KompTi** [wazuh/wazuh#15489] [Q&A]
Hi everyone, have a nice day

We are customizing the wazuh rules to detect **Lockbit** ransomware because, recently this **RaaS** affected all the systems of Chilean Judiciary. Actually the [rules in wazuh](https://wazuh.com/blog/detecting-lockbit-3-0-ransomware-with-wazuh/) has no connection with the **Lockbit 3.0** attacks reported in your blog website.

Please, take a few minute to read the [CSIRT report](https://www.csirt.gob.cl/noticias/actualizacion-alerta-poder-judicial/)
They explain and bring important details to detect it

So, is very important develope a rule that avalible for enterprise and government institutions on the wazuh repositories

Best regards for you!

**Comment by thony4uu** [comment]
Hello KompTi

Thank you for using Wazuh and bringing to our attention this development

I have analyzed the report in the link you shared relating to the attack on Chilean Judiciary and can confirm that this attack has no connection with the latest Lockbit 3.0 attacks that has been reported thus far.

VirusTotal could not detect 2 of the IOCs in the report (123.cmd and lbb.dll) while injector.exe that was detected had no connection to ransomware activity as per the report and comments.

I have tried to obtain the files to test this attack however the files are not available on various malware repository websites

I would be glad to assist if you have further information to share to aid my analysis however the current information related to this attack does not really portray the attack pattern of Lockbit 3.0 seen thus far.

Also Wazuh can be configured to detect the attack by adding the IOCs to detection rules.

For example, you can configure FIM to monitor the directories on the system to detect thes files. See example below 
Configure FIM to monitor download folder by adding the configuration in the Ossec.conf file on the endpoint 
<syscheck>  
<directories check_all="yes" realtime="yes">C:\Users\*\Downloads</directories>
</syscheck>

Add  the following rules to /var/ossec/etc/rules/local_rules.xml file on the Wazuh server
<group name="syscheck,malware,">

  <rule id="100008" level="10">
    <if_sid>554</if_sid>
    <field name="sha256">385D4FBD3A3AAB0CFE109F0EB2626863B91EC02CEAA4C3E4C104F63CA6DC0C93</field>
    <field name="sha256">0DA8CFBA645951768B16DB7707ED2C75BBBB2E396E7DA6DA71A9DCA6EF35DF17</field>
    <field name="sha256">5EF365469867CB30F5094D668E596819BF7163411E2A92CC1D7BD117FCD11BFE</field>
    <description>Malware Hash Match: Lockbit Ransomware Executable Detected</description>
  </rule>

Kindly restart Wazuh server after adding the rules. 

Also as demonstrated in the  [blog post](https://wazuh.com/blog/detecting-lockbit-3-0-ransomware-with-wazuh/), you can define leverage VirusTotal, YARA and define your own custom rules to detect the ransomware 

Please let me know if you have further queries

Best Regards

**Reply by muhyuddin** [reply]
Can you please provide this alert sample for directly matching ioc hashes in custom rules ? Have been trying to play with FIM for triggering alert on hash match, but no success. Thanks.

**Comment by KompTi** [comment]
Thank a lot for your answer!
